public/app/core/components/sidemenu/SideMenuDropDown.tsx

```diff
@@ -21,15 +21,19 @@ const SideMenuDropDown = ({
   reverseDirection = false,
   subtitleText,
 }: Props) => {
+  const resetButtonStyles = css`
+    background-color: transparent;
+    font-size: inherit;
+  `;
   const headerContent = <span className="sidemenu-item-text">{headerText}</span>;
   const header = headerUrl ? (
     <Link href={headerUrl} onClick={onHeaderClick} className="side-menu-header-link">
       {headerContent}
     </Link>
   ) : (
-    <a onClick={onHeaderClick} className="side-menu-header-link">
+    <button onClick={onHeaderClick} className={cx(resetButtonStyles, 'side-menu-header-link')}>
```

Comment:
This seems hacky, we should update the side-menu-header-link style (or migrate it to emotion) not just add some emotion styles on top